packages/vats/src/bridge.js

```diff
@@ -1,90 +1,183 @@
 // @ts-check
 
-import { makeScalarMapStore } from '@agoric/store';
+import { M } from '@agoric/store';
 import '@agoric/store/exported.js';
-import { Fail } from '@agoric/assert';
-import { Far } from '@endo/far';
-import { makeWithQueue } from '@agoric/internal/src/queue.js';
+import { E } from '@endo/far';
+import './core/types.js';
 
-/**
- * @template T
- * @typedef {'Device' & { __deviceType__: T }} Device
- */
+const { Fail, details: X } = assert;
+
+export const BridgeHandlerI = M.interface('BridgeHandler', {
+  fromBridge: M.call(M.any()).returns(M.promise()),
+});
+
+export const BridgeScopedManagerI = M.interface('ScopedBridgeManager', {
+  fromBridge: M.call(M.any()).returns(M.promise()),
+  toBridge: M.call(M.any()).returns(M.promise()),
+  setHandler: M.call(M.remotable('BridgeHandler')).returns(),
+});
+
+export const BridgeManagerI = M.interface('BridgeManager', {
+  register: M.call(M.string())
+    .optional(BridgeHandlerI)
+    .returns(M.remotable('BridgeScopedManager')),
+});
+
+const BridgeManagerKitI = harden({
```

Comment:
```suggestion
const BridgeManagerIKit = harden({
```
(cf. https://github.com/Agoric/agoric-sdk/pull/5997#discussion_r1044581388 and https://github.com/Agoric/agoric-sdk/pull/5997#pullrequestreview-1212394249 and [repo:**Agoric/agoric-sdk** iKit](https://github.com/search?q=repo%3AAgoric%2Fagoric-sdk+iKit))